src/v8/isolate.rs

```diff
@@ -237,6 +254,21 @@ impl V8Isolate {
     pub fn cancel_terminate_execution(&self) {
         unsafe { v8_CancelTerminateExecution(self.inner_isolate) }
     }
+
+    /// Returns a raw pointer to a [v8_isolate].
+    pub fn get_raw(&self) -> *mut v8_isolate {
+        self.inner_isolate
+    }
+
+    /// Returns the unique ID of this isolate.
+    pub fn get_id(&self) -> Option<IsolateId> {
```

Comment:
Shouldn't this be `pub(crate)`? Isn't the ID an internal implementation detail?

Reply:
Also, could it be that we will have an isolate without an ID? If its not possible maybe we should assert it?

Reply:
> I'd move this assertion the the caller's side. We shouldn't decide for the caller what he should do in case something goes wrong.

I think that the question should be whether or not this can happened, if it can not then why providing an API that will require the caller to handle the error?

And what about the `pub(crate)` suggestion?

Reply:
Ok I am convinced, lets keep it.